sporks.json

```diff
@@ -1,950 +1,995 @@
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.14.0",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.14.0",
-      "access-node": ""
-    },
-    {
-      "spork-id": "Devnet18",
-      "spork-date": "2020-12-11T16:00:00Z",
-      "root-height": "17756122",
-      "root-parent-id": "52ace0d0c8d4e574213fe98e19a5043f215bee992659bd1ef35c5758acf54d1b",
-      "root-state-commit": "b1412c9d453b1c10d3ace6111e00ac804d564ba3d3295002bedb077685c7da73",
-      "git-commit-hash": "115ccee9cae920ee5bc309537cf43a9b152a1cf9",
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.13.0",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.13.0",
-      "access-node": ""
-    },
-    {
-      "spork-id": "Devnet17",
-      "spork-date": "2020-11-27T16:00:00Z",
-      "root-height": "16483518",
-      "root-parent-id": "96d452f9d2a15fafab720f1809fc490b019b23e42c885590e62085e48b8c2b6b",
-      "root-state-commit": "0b45f6055a1f09b413d098997cc0e7c9a0ef19eac8b4d294085b0e2f436c6bff",
-      "git-commit-hash": "ef54713595d0fe1e0bb4b14e9469c4a64dfaf6e7",
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.12.1",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.12.1",
-      "access-node": ""
+  "networks" : [
+    {
+      "name": "mainnet",
+      "sporks": [
+        {
+          "id": 14,
+          "name": "Mainnet-14",
+          "spork-date": "2021-10-06T16:00:00Z",
+          "root-height": "19050753",
+          "root-parent-id": "ac4dbf344ce96e39e15081f1dc3fbbf6dc80532e402de9a57af847d3b35df596",
+          "root-state-commit": "641eb088e3ce1a01ff56df2d3a14372c65a7fef44c08799eb92cd7759d1d1d2a",
+          "git-commit-hash": "f019c1dbd778ce9f92dea61349ca36003678a9ad",
+          "state-artefacts" : {
```

Comment:
interesting that we decided to go with this spelling of artefact

Reply:
Happy to change it LMK